plugins/language_caddyfile.lua

```diff
@@ -2,7 +2,7 @@
 local syntax = require "core.syntax"
 
 syntax.add {
-  files = { "Caddyfile" },
+  files = { "/Caddyfile" },
```

Comment:
Should maybe add `$`?

Reply:
good catch, done on all the ones you spotted